tools/clean.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/zsh
+
```

Comment:
```suggestion
set -e
```

we must add this in every script so no error is ignored automatically.